source/bringauto/modules/ModuleManagerLibraryHandler.cpp

```diff
@@ -113,4 +120,17 @@ void ModuleManagerLibraryHandler::deallocate(struct buffer *buffer){
 	deallocate_(buffer);
 }
 
-}
\ No newline at end of file
+bringauto::modules::Buffer ModuleManagerLibraryHandler::constructBuffer(std::size_t size) {
+	struct ::buffer buff;
+	buff.size_in_bytes = size;
+	if(allocate(&buff, size) != OK) {
+		throw std::bad_alloc();
```

Comment:
pls std::bad_alloc {} (I know the () are used in many places but {} implies the struct....)

Reply:
Changed braces